src/main/java/by/yurachel/web_app/repository/PhoneRepository.java

```diff
@@ -1,33 +1,42 @@
 package by.yurachel.web_app.repository;
 
+import by.yurachel.web_app.dao.PhoneListDAO;
 import by.yurachel.web_app.entity.Phone;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
-import java.util.ArrayList;
-import java.util.List;
+import java.util.*;
 
 public class PhoneRepository {
```

Comment:
This is redundant class as we added DAO and entities are now retrieved from the DB